### DISCUSSION TITLE
トラックボールなし側のキースイッチが浮く

### DISCUSSION THREAD

**Discussion by momocus** [Yowkees/keyball#386] [Q&A] [answered]
### 背景

トラックボールなし側のほうにキースイッチを刺すと、最後まで刺さらずに浮きます。

←トラックボールあり　トラックボールなし→

横から見た写真 

![IMG_2377](https://github.com/Yowkees/keyball/assets/6294782/dd999394-6d11-41b2-8b21-a07c6e4a2fe3)

上から見た写真

![IMG_2376](https://github.com/Yowkees/keyball/assets/6294782/08929879-bab2-4aec-8841-fb0859a89d10)

### 環境

- キット: keyball44
- キースイッチ: 下記キースイッチで試し、全て同じように浮きました。
    - Durock White Lotus / Light Tactile
    - Gateron スイッチ 茶軸
    - Gateron スイッチ 緑軸
 
### 仮説

トラックボールなし側トップPCBの穴が(極端に描くと)下図のようになっています。これにより、縦幅が足りずにキースイッチが刺さらないようです。

![IMG_2379](https://github.com/Yowkees/keyball/assets/6294782/2d5cc8f5-1975-4020-969c-f2dab195e8dc)

### お願い

- 製造上の不良品ということでしたら、トップPCBの交換をお願いしたいです。
- できれば: 製造不良の起こりにくい公差設計になっているかをご確認下さい。

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

キースイッチ差し込み口が狭めのトッププレートをお届けしてしまったようでご迷惑をおかけして大変申し訳ありません。
交換等についてご相談したいので、お手数ですが下記白銀ラボウェブショップのお問合せよりご連絡いただけますでしょうか。
https://shirogane-lab.net/inquiry

以上、よろしくお願いいたします。

**Reply by momocus** [reply]
本日受領したトップPCBで、無事組み付けできました。ご対応ありがとうございました。

![image](https://github.com/Yowkees/keyball/assets/6294782/e3790a82-36bb-4782-8e18-09055aadfb51)
